app/lib/aws_util.rb

```diff
@@ -0,0 +1,6 @@
+class AwsUtil
+  def self.get_cloudwatch_url(log_group, log_stream)
+    "https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2" \
```

Comment:
I feel like `us-west-2` should be in a global env var, but I don't see other usages with one. Or you could have one in this file?